src/firebase_rtdb_client.cpp

```diff
@@ -0,0 +1,21 @@
+#include "firebase_rtdb_client.hpp"
+
+void FirebaseRTDBClient::begin()
```

Comment:
```suggestion
void FirebaseRTDBClient::setup()
```

setupに変えてビルドできたのでかえよう